doc/faq.md

```diff
@@ -28,6 +28,14 @@ That said, it is definitely possible to have write patterns that defeat this. De
 
 Theoretically, definitely. In Git, for example, the concept of "shallow clones" exists, and we could do something similar in Noms. This has not been implemented yet.
 
+### Is this another one of those NoSQL database thingies?
+
+It depends on the specific features and/or functionality you use to define `NoSQL`. Does Noms support similar type of data? Yes. Does Noms support a flexible schema? Yes.  Does Noms "compromise consistency"? No. Is Noms “eventually consistent”? No, it is always consistent.
```

Comment:
The thing is that usually when people are asking about this they are thinking about something like mongo. Noms is not a distributed database, so these questions kind of don't make sense. When people talk about 'eventual consistency' they are talking about a multimaster situation with a distributed database -- you have writes at multiple servers and they need to resolve somehow. This is asking a question at a higher level of abstraction than we are currently playing at, so it's kind of hard to answer.